packages/lesswrong/lib/collections/electionVotes/schema.ts

```diff
@@ -0,0 +1,120 @@
+import { foreignKeyField } from "../../utils/schemaUtils";
+import { userOwns } from "../../vulcan-users";
+
+const validateVote = ({data}: {data: Partial<DbElectionVote>}) => {
+  if (data.vote && typeof data.vote !== 'object') {
+    throw new Error("Invalid vote value");
+  }
+  for (let key in data.vote) {
+    if (typeof data.vote[key] !== 'number' && data.vote[key] !== null) {
+      throw new Error("Invalid vote value");
+    }
+    if (data.vote[key] !== null && data.vote[key] < 0) {
```

Comment:
It might be overly cautious, but I'd normally though in a check for `Number.isFinite(data.vote[key])` for this kind of thing too just to weed out any NaNs or other funky stuff. (Also not necessary in this PR, but I thought I'd mention it).